Gemfile.lock

```diff
@@ -1,8 +1,8 @@
 PATH
   remote: .
   specs:
-    squash_repeater (0.1.12)
-      backburner (~> 1.0)
+    squash_repeater (0.1.13)
+      backburner (= 1.0)
```

Comment:
Because I haven't had time to review the changes to backburner between 1.0 & 1.3, and bundle install wants to install 1.3 if I bundle with `-> 1.3` 

:(

Hey Will!